source/frontend/src/hooks/repositories/useRequisitionCompensationRepository.ts

```diff
@@ -168,6 +173,93 @@ export const useCompensationRequisitionRepository = () => {
     invoke: false,
   });
 
+  const getCompensationRequisitionAtTime = useApiRequestWrapper<
+    (
+      compensationId: number,
+      time: string,
+    ) => Promise<AxiosResponse<ApiGen_Concepts_CompensationRequisition, any>>
+  >({
+    requestFunction: useCallback(
+      async (compensationId: number, time: string) =>
+        await getCompensationRequisitionAtTimeApi(compensationId, time),
+      [],
+    ),
+    requestName: 'getCompensationRequiistionAtTime',
+    onSuccess: useAxiosSuccessHandler(),
+    onError: useAxiosErrorHandler('Faied to get compensation requisition finalized'),
```

Comment:
nit(s):
finalized vs. historical (consistency)
Faied - Failed

Reply:
Maybe the team should adopt a spell check plugin? there must be one that is smart enough to only check comments and user-visible text.

Reply:
😅